cmake/projects/libuv/hunter.cmake

```diff
@@ -7,6 +7,17 @@ include(hunter_download)
 include(hunter_pick_scheme)
 include(hunter_report_broken_package)
 
+hunter_add_version(
```

Comment:
we changed to adding new versions at the bottom of the list. Please move the new version one to the bottom